tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
```

Comment:
If we are on a spree of dropping files, I don't think any of us are using this one,  and it doesn't seem to support all the supported python versions

Reply:
Let's merge updates first, and then drop files. I mentioned one in my review, because it is added in this PR.

Reply:
I have created a separate [PR](https://github.com/scylladb/python-driver/pull/368) for it.